cloudformation/policies/parallelcluster-policies.yaml

```diff
@@ -743,32 +746,21 @@ Resources:
               StringLike:
                 iam:AWSServiceName:
                   - imagebuilder.amazonaws.com
-
-  ParallelClusterDeleteImageManagedPolicy:
-    Type: AWS::IAM::ManagedPolicy
-    Properties:
-      ManagedPolicyName: !Sub
-        - ${IAMRoleAndPolicyPrefix}ParallelClusterDeleteImageManagedPolicy-${StackIdSuffix}
-        - { StackIdSuffix: !Select [ 0, !Split [ '-', !Select [ 2, !Split [ '/', !Ref 'AWS::StackId' ] ] ] ] }
-      Description: Managed policy to execute pcluster delete-image command without IAM permission
-      PolicyDocument:
-        Version: '2012-10-17'
-        Statement:
-          - Sid: EC2
+          - Sid: EC2DeleteImage
             Effect: Allow
             Action:
               - ec2:DeregisterImage
               - ec2:DescribeImages
```

Comment:
Non-blocking comment and can be addressed in a different PR

We are giving the same access like `ec2:DescribeImages` for multiple places like `Sid: EC2DeleteImage, Sid: EC2ListImages and Sid: EC2BuildImage` We might want to combine them later so that we dont run into Policy Doc length issue.

Reply:
I definitively agree with you. We will address in a follow up PR. Better to not play too much with permissions last minute :-)